### DISCUSSION TITLE
SOP Compliance: Tracking and Training Insights

### DISCUSSION THREAD

**Discussion by Mau-FTL** [GitbookIO/community#363] [0. General bugs and questions]
I've used this platform for personal notes and How-To guides for technical tasks. I really like how quick it is for me to create documentation. I'm considering introducing it to my organization for SOP compliance. 

**Does GitBook offer a feature to confirm that a user has read and agreed to the SOP?** 

It would guarantee that every member in my organization is on the same page and also enables managers to identify and address training needs efficiently, ensuring seamless operations and adherence to established protocols.

**Comment by aikoii** [comment]
Hey @Mau-FLA! Thanks for the GitBook love 💙! While we don't have a built-in feature for SOP acknowledgment, embedding feedback forms or linking to external platforms for confirmation could be a workaround here.
